js/test.js

```diff
@@ -36,72 +25,93 @@ if (window.location.pathname.endsWith ('quiz.html')) {
   } else {
     alert('Your answer must be a yes or no.');
   }
+ }
+ quizHoodFunc()
 
-  var quizBS = prompt('Does Matthew have a Bachelor of Science in Engineering, Yes or No?');
-  console.log('user\'s answer was ' + quizBS);
-  var lowerCaseQuizBS = quizBS.toLowerCase();
-  if(lowerCaseQuizBS === 'no'|| lowerCaseQuizBS === 'n') {
-    alert('Correct, Matthew has a BS in International Studies.');
-    score++;
-  } else if(lowerCaseQuizBS === 'yes'|| lowerCaseQuizBS === 'y') {
-    alert('Negative, Matthew has a BS in International Studies.');
-  } else {
-    alert('Your answer must be a yes or no.');
-  }
-
-  var quizCode = prompt('Is Matthew in school at Code Fellows, Yes or No?');
-  console.log('user\'s answer was ' + quizCode);
-  var lowerCaseQuizCode = quizCode.toLowerCase();
-  if(lowerCaseQuizCode === 'no'|| lowerCaseQuizCode === 'n') {
-    alert('Negative, Matthew is in school at Code Fellows.');
-  } else if(lowerCaseQuizCode === 'yes'|| lowerCaseQuizCode === 'y') {
-    alert('Correct, Matthew is in school at Code Fellows.');
-    score++;
-  } else {
-    alert('Your answer must be a yes or no.');
+ function quizBsFunc() {
```

Comment:
You have linter indentation errors on your functions. Fix those up! Let me know if you have questions.